.github/workflows/check.yaml

```diff
@@ -105,5 +105,8 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install 'tox<5'
 
-      - name: Run functional tests
-        run: tox -e func
+      - name: Run func tests
+        run: |
+          export TEST_SNAP="$(pwd)/$(ls | grep '.*_.*\.snap$')"
+          echo "$TEST_SNAP"
```

Comment:
Maybe consider removing `echo` from the command, since you can also see this from CI's output